app/code/community/Sandfox/GeoIP/Model/Country.php

```diff
@@ -43,6 +44,16 @@ public function isCountryAllowed($country = '')
         }
     }
 
+    public function isDefaultCountry($country = '')
```

Comment:
you should implement two different methods. isDefaultCountry is no proper name, if you pass a country in an isCountry is no proper name if you don't pass a country in :-)